src/fidesops/ops/schemas/privacy_request.py

```diff
@@ -107,6 +108,25 @@ class ExecutionLogDetailResponse(ExecutionLogResponse):
     dataset_name: Optional[str]
 
 
+class ExecutionAndAuditLogResponse(BaseSchema):
+    """Schema for the combined ExecutionLogs and Audit Logs
+    associated with a PrivacyRequest"""
+
+    collection_name: Optional[str]
+    fields_affected: Optional[List[FieldsAffectedResponse]]
+    message: Optional[str]
+    action_type: Optional[ActionType]
+    status: Optional[Union[ExecutionLogStatus, AuditLogAction]]
+    updated_at: Optional[datetime]
+    user_id: Optional[str]
+
+    class Config:
+        """Set orm_mode and allow population by field name"""
+
+        use_enum_values = True
+        allow_population_by_field_name = True
+
```

Comment:
I originally extended the ExecutionLogResponse schema above and just overrode fields to make them optional or change their type but mypy didn't like this, so I made a new one.  I also `allow_population_by_field_name` here because it is no longer ExecutionLogs that are getting serialized with this, it's a union query of audit logs and execution logs.